src/mudlet.cpp

```diff
@@ -324,7 +324,7 @@ mudlet::mudlet()
 
     mpActionDiscord = new QAction(tr("Open Discord"), this);
```

Comment:
Do we want to add a comment along the lines that "Discord" here is an on-line chat/communications application - and should not be translated (unless they use a different name in other languages?)...

Reply:
I think people will know what Discord is, it's ok. If they don't, they won't know how to translate it anyhow..